src/main/java/com/google/sps/utility/GeocodingResultUtility.java

```diff
@@ -18,57 +18,79 @@
 import com.google.maps.model.GeocodingResult;
 import com.google.maps.model.LatLng;
 import com.google.maps.model.PlaceType;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Optional;
 
 /** Utility class to extract data from GeocodingResult objects. */
 public class GeocodingResultUtility {
   /**
-   * Parses for coordinates in a resulting call to the Geocoding API.
+   * Parses for the first coordinate found in a resulting call to the Geocoding API.
    *
    * @param result A GeocodingResult returned from the Geocoding API.
    * @return A LatLng representing coordinates.
    */
-  public static LatLng getCoordinates(GeocodingResult result) {
-    return result.geometry.location;
+  public static LatLng getCoordinates(List<GeocodingResult> results) {
+    for (GeocodingResult result : results) {
+      List<AddressType> types = Arrays.asList(result.types);
+      for (AddressType type : types) {
```

Comment:
nit: maybe iterate directly on `result.types`?

Reply:
Yeah I agree, there's no reason to store the types variable. Thanks!

Reply:
`RESOLVED`